Frontend/src/app/components/product-list/product-list.component.spec.ts

```diff
@@ -1,20 +1,20 @@
 import {ComponentFixture, TestBed} from '@angular/core/testing';
 
 import {ProductListComponent} from './product-list.component';
-import {FooterComponent} from "../footer/footer.component";
-import {HeaderComponent} from "../header/header.component";
-import {RouterTestingModule} from "@angular/router/testing";
-import {ApiService} from "../../services/api.service";
-import {AbstractMockObservableService} from "../../mocks/mock.service";
-import {Router} from "@angular/router";
+import {FooterComponent} from '../footer/footer.component';
+import {HeaderComponent} from '../header/header.component';
+import {RouterTestingModule} from '@angular/router/testing';
+import {ApiService} from '../../services/api.service';
+import {AbstractMockObservableService} from '../../mocks/mock.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mueller-Patrick/Betterzon/pullRequest?prid=7401082)
```suggestion
import {AbstractMockObservableService} from "../../mocks/mock.service";
```